modules/cnv-subscribing-to-hco-catalog.adoc

```diff
@@ -12,28 +12,36 @@ namespace access to the {CNVProductName} Operators.
 
 .Prerequisites
 
-* `OperatorGroup` and `CatalogSource` Custom Resource Definition objects (CRDs),
-both created in the `openshift-cnv` namespace
+* Create a namespace called `openshift-cnv`.
 
 .Procedure
 
-. Open a browser window and navigate to the {product-title} web console.
+. Open a browser window and log in to the {product-title} web console.
 
-. Navigate in the web console to the *Operators → OperatorHub* page.
+. Navigate to the *Operators → OperatorHub* page.
 
-. Scroll or type a keyword into the *Filter by keyword* box to find the KubeVirt
-HyperConverged Cluster Operator, then click on it.
+. Scroll or type a keyword into the *Filter by keyword* box to find the
+*KubeVirt HyperConverged Cluster Operator*, then click on it.
 
 . Read the information about the Operator and click *Install*.
 
 . On the *Create Operator Subscription* page:
-.. Select one of the following:
+.. Select *A specific namespace on the cluster* from the *Installation Mode*
```

Comment:
There are two of them.

Reply:
Thanks! Then you need to present the two options in an unordered list and make it clear that you do one of them.

Reply:
I'm a bit confused. My understanding is that we don't want the customer to think that they could choose either option -- they need to pick "A specific namespace on the cluster." I presented both options in the note in order to explain them/give context, because both options do appear in the UI. Is that sufficient or should I rewrite the step?